docs/customized-components/MenuList.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+/* eslint-disable import/no-unresolved */
```

Comment:
interesting :D please run `npm run build` and check if this works after build

Reply:
tested locally and works fine